test/e2e/operation/pipeline/src/test/resources/env/it-env.properties

```diff
@@ -25,8 +25,11 @@ pipeline.it.docker.opengauss.version=
 # pipeline.it.native.database=mysql or postgresql or opengauss
 pipeline.it.native.database=
 pipeline.it.native.mysql.username=root
-pipeline.it.native.mysql.password=Root@123
+pipeline.it.native.mysql.password=root
 pipeline.it.native.mysql.port=3306
+pipeline.it.native.mariadb.username=root
+pipeline.it.native.mariadb.password=root
+pipeline.it.native.mariadb.port=3306
```

Comment:
1, MySQL password should be recovered

2, MariaDB password could use similar password